plugins/PostProcessingPlugin/scripts/PauseAtHeight.py

```diff
@@ -507,7 +507,16 @@ def execute(self, data: List[str]) -> List[str]:
                     else:
                         Logger.log("w", "No previous feedrate found in gcode, feedrate for next layer(s) might be incorrect")
 
-                    prepend_gcode += self.putValue(M = 82) + " ; switch back to absolute E values\n"
+                    extrusion_mode_string = "absolute"
+                    extrusion_mode_numeric = 82
+
+                    extruders = list(Application.getInstance().getGlobalContainerStack().extruders.values())
+                    relative_extrusion = extruders[0].getProperty("relative_extrusion", "value")
```

Comment:
`relative_extrusion` is a global setting. It cannot be changed in an extruder. If it were, getting it from the first extruder would be incorrect anyway (since you don't know which extruder was active during the pause). Luckily since it's a global setting the solution is simple: Just get it from the global stack:

```suggestion
                    relative_extrusion = Application.getInstance().getGlobalContainerStack().getProperty("relative_extrusion", "value")
```